app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<h2 class='logTitle' >Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+      <%= f.text_field :name, autofocus: true,  placeholder: "name",autocomplete: "name", class: :signUpName %>
+  </div>
+
+  <div class="field">
+    <%= f.email_field :email, autofocus: true,  placeholder: "email",autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+      <%= f.text_area :bio, autofocus: true,  placeholder: "bio",autocomplete: "bio" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :posts_counter, class: "label" %><br />
+    <%= f.number_field :posts_counter, value: 0, autofocus: true, placeholder: "posts_counter",autocomplete: "posts_counter" %>
+  </div>
+
+  <div class="field">
+     <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
```

Comment:
- [ ] Kindly make sure your sign-in and sign-up pages matches  the design wireframe, you can center ypour form to the center of the page, increase size of the button 👍🏿 



Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![blog_login (2)](https://github.com/faisalakb/R_blog_app/assets/69638013/488062aa-fefd-4eae-b6cf-83523260bc27) |  ![Screenshot_239](https://github.com/faisalakb/R_blog_app/assets/69638013/00d5a8ed-b5d3-4575-9cef-93dac85225aa)